### PR TITLE
New Sites: Add a "Day of Event" page to new sites

### PR DESCRIPTION
**Pull request by ryelle**
This creates a new page that will use the Day of Event template on new sites. It includes the Live Schedule block from #243, and the live Latest Posts block from #250. Now that all this content is blocks, organizers can fully control the information & style of this page.

**To test**

- Create a new WordCamp site
- The new site should pre-populate with a page called "Day of Event", with useful information for during the camp (date, location, live schedule, latest posts).

### CODE REVIEW

**Comment by ryelle** [comment]
This will wait on #243 & #250 – once those are merged, we can use the new blocks to replicate the existing Day of Event template, and remove the duplicate JS code.

**Comment by ryelle** [comment]
Live "Latest Posts" block requires the Gutenberg plugin to be active.

**Comment by ryelle** [comment]
Now that these blocks are enabled for everyone, this is _actually_ ready for review :)

File: public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-new-site.php
Comment (coreymckrill) [inline public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-new-site.php]: Should this be `draft` instead of `publish`, since the page is only intended to be used near the time of the actual event?

**Review by coreymckrill — APPROVED** [review]
👍 Tested locally

File: public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-new-site.php
Comment (ryelle) [reply public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-new-site.php]: oh, that's a good idea 👍

**Pull request merged**
